.vib/linux-install/goss.yaml

```diff
@@ -1,5 +1,5 @@
 command:
   install-proxy:
-    exec: /scripts/tests.sh
+    exec: /tmp/extracted/scripts/tests.sh
```

Comment:
We fixed a bug and now you can use relative paths, `./scripts/tests.sh` should work.